charts/flux2/values.yaml

```diff
@@ -134,6 +134,15 @@ kustomizecontroller:
   nodeSelector: {}
   affinity: {}
   tolerations: []
+  # Allows you to load environment variables from kubernetes secret or config map
```

Comment:
If you use
   `# --  some text`
It will be shown in the README as value description